### PR TITLE
Adds RTCOutboundRtpStreamStats.rid

### LINKED ISSUE #395: Add RTCOutboundRtpStreamStats.rid

### PR DESCRIPTION
**Pull request by henbos**
Fixes #395

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -1918,6 +1919,17 @@ <h2>
...
                 </p>
               </dd>
+              <dt>
+                <dfn><code>rid</code></dfn> of type <span class="idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  Exposes the <a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtpcodingparameters-rid">rid</a>
+                  encoding parameter of this RTP stream if it has been set, otherwise it is
+                  undefined. If set this value will be present regardless if the RID RTP header
```
Comment (vr000m) [inline webrtc-stats.html]: The value will be present even if the RID RTP header extension has not been negotiated.

**Comment by henbos** [comment]
Before merging we need to decide if we want encoding-parameters as a separate object referenced by the outbound-rtp or if we want all per-layer metrics to be part of the RTCOutboundRtpStreamStats

**Comment by henbos** [comment]
This will probably become obsolete by #495

**Comment by vr000m** [comment]
OBE by #495

**Pull request merged**